examples/Components/misc/ReadState.scn

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
```

Comment:
for the future, use `git move` to rename a file, so that you keep the history of the file